src/officehours_ui/templates/base.html

```diff
@@ -86,6 +86,17 @@
   <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
   <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script>
+    const logoutButton = document.querySelector('#logout-button');
+    logoutButton.addEventListener('click', () => {
+      window.localStorage.setItem("loggedIn", "false");
+    });
+    window.addEventListener('storage', (event) => {
+      if (document.cookie.indexOf('JSESSIONID') === -1) {
+        window.location.href = "/accounts/login/";
+      }
```

Comment:
Can you say a little more what's going on with `document.cookie.indexOf('JSESSIONID')`? It looks like`JSESSIONID` is a Java thing, which wouldn't be present here in this application. The Django session cookie is HTTP Only, so it's not accessible in the frontend. So this condition always passes since it's never found.

It would be nice to check for the existence of the Django session cookie, but I haven't found a way to do that yet without compromising security.

We **could** just remove this check entirely. I'm not sure what other option we have. Let me know if I'm missing something here.

Reply:
I'm wondering if we want to use the `event` to make sure this is the local storage change we're looking for, and not someone just manually adding stuff. Not that there's much risk in that, it's an edge case. But I could also see someone in the future trying to add some other usage of local storage to the application and this becoming an issue.

So referring to the [MDN docs here](https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event), I think we could do something like this to keep this behavior isolated?
```
      if (event.key === 'loggedIn' && event.newValue === 'false' && event.oldValue === 'true') {
        window.location.href = '/accounts/login';
      }
```

Reply:
Yeah, that's interesting. The only cookie set by the app is `sessionid` but this is HttpOnly so it can't be checked. It does look like it was working though locally even though there is no JSESSIONID. 

I think if there's another way to work around it without checking the presence of the cookie it _might_ be better as I'm not sure if this would work with weblogin without some testing.